using_openshift/output_deps.adoc

```diff
@@ -0,0 +1,54 @@
+= Dependency tree generation
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title: 
+
+toc::[]
+
+== Overview
+`build-chain` is an experimental command-line tool for OpenShift v3. This tool will start at the imagerepo:tag specified and determine what Builds will be triggered (due to ImageChangeTriggers on the BuildConfig) when this imagerepo:tag is updated. Further, it analyzes the output of those Builds to determine if they in turn update another imagerepo:tag. If they do, the tool continues to follow the dependency tree. It then outputs the total graph of what imagerepo:tags will be impacted by an update to the top level imagerepo:tag. The default output is set to JSON but the user can also choose between DOT and AST.
+ 
+ 
+== Examples
+
+
+Build dependency tree for the specified image repository with the specified tag in the current namespace and output it in JSON
+----
+$ openshift ex build-chain [image-repository]:[tag]
+----
+
+Build and output the dependency tree for the specified image repository in 'testing' namespace using DOT syntax
```

Comment:
this will use the "latest" tag since none is specified, right?  should indicate that.